cmd/gafFilter/gafFilter.go

```diff
@@ -0,0 +1,76 @@
+// Command Group:
+
+// Output a subset of a gaf file without "NOT" entries in qualifier field
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fileio"
+	"github.com/vertgenlab/gonomics/ontology/gaf"
+	"log"
+	"strings"
+)
+
+type Settings struct {
+	InFile    string
+	OutFile   string
+	RemoveNot bool
+}
+
+func gafFilter(s Settings) {
+	var pass bool = false
```

Comment:
style: if you set pass to true at the begining of each loop below, do you have to explicitely define it as false here?

Reply:
Probably not, good point